### PR TITLE
Negative years

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[UI-config kann keine negativen Start-Jahre enthalten.](https://trello.com/c/jojIIr5X/1101-ui-config-kann-keine-negativen-start-jahre-enthalten)

File: README.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ You can override any of these default settings by specifying them.
     }
 ```
 
+Note:
+
```
Comment (ChristianBeilschmidt) [inline README.md:113]: When momentjs tries to parse such a date with 4 digits (negative), then it seems to return `undefined`.
Could we add an error to the console in this case when we try to load it from the config?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (michaelmattig) [reply README.md:113]: 630d687c5824ebc432e62e84ff15ab97c55144a5

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**